views/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+
+<!--
+（＾・ﻌ・＾✿）
+
+    photo_box images are from: https://hyperallergic.com/239683/how-would-you-draw-the-internet/
+
+            （ฅ＾・ﻌ・＾）ฅ 
+-->
+
+<head>
+    <meta charset="utf-8">
+    <link href="../styles/normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Space+Mono" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+    <link href="../styles/main.css" rel="stylesheet">
+    <title>guillermina</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+
+        </nav>
+    </header>
+    <section class="header_box">
+
+    </section>
+    <section class="main_box">
+        <h2>hi i'm guillermina muro,</h2>
+        <section class="main_info">
+            <h4>currently:</h4>
+            <ul>
+                <li>a student at <a href="https://http://adadevelopersacademy.org/">Ada Developers Academy</a> in Seattle, WA</li>
+                <li>probably applying sunscreen</li>
+            </ul>
+        </section>
+
+
+    </section>
+
+
+    <section class="photo_box">
+        <article id="index">
```

Comment:
this might actually be a good use case for not using `<article>` but instead using `<div>`